features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/logic/flot.app-by-popularity-over-time.js

```diff
@@ -23,7 +23,7 @@ function drawGraphs() {
                    'endDate': to
                },
                success: function (response) {
-                   parsedResponse = JSON.parse(response);
+                   parsedResponse = response;
                    drawPopularityOverTime(parsedResponse);
```

Comment:
Are we using parsedResponse variable outside of the success function? If not we do not need L26.